pokerole_irl/worlds_app/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from .views import SettingListView, SettingCreateView, SettingView, CharacterView, CharacterCreateView
+
+urlpatterns = [
+    path('setting_list/', SettingListView.as_view(), name='setting-list'),
+    path('setting_create/', SettingCreateView.as_view(), name='setting-create'),
+    path('setting/<int:pk>', SettingView.as_view(), name='setting'),
+    path('character_create/', CharacterCreateView.as_view(), name='character-create'),
```

Comment:
Here we should use "character/add/"

Reply:
This should probably actually be on the setting URL, since you create characters for a setting. i. e. "<int:setting_pk>/character/add/"